recipes/fleur/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "fleur" %}
+{% set version = "6.0" %}
+{% set version_long = "MaX-R" + version %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://iffgit.fz-juelich.de/{{ name }}/{{ name }}/-/archive/{{ version_long }}/{{ name }}-{{ version_long }}.tar.gz
+  sha256: b022793360c198e18f269541152a04189990c0bf8326939f0f24bb8ca2adf757
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - make
+    - git
+    - doxygen
+  host:
+    - libblas
+    - liblapack
+    - libgomp
+    - openmpi
+    - scalapack
+    - libxml2
+    - fftw
+  run:
+    - openmpi
+    - scalapack
+
+test:
+  commands:
+    - inpgen -h    
+
```

```suggestion
test:
  commands:
    - inpgen -h    
    - test -f $PREFIX/bin/fleur_MPI  # [unix]
```
Are you aware that you are shipping this binary? If so, test that it exists.